src/components/Skeleton/Skeleton.module.css

```diff
@@ -0,0 +1,35 @@
+/*------------------------------------*\
+    # SKELETON
+\*------------------------------------*/
+
+@keyframes bg-wipe {
+  0% {
+    background-position: 0% 0%;
+  }
+  100% {
+    background-position: 100% 0%;
+  }
+}
+
+/**
+ * Skeleton
+ */
+.skeleton {
+  border-radius: var(--eds-border-radius-lg);
+  background-color: var(--eds-theme-color-background-neutral-medium);
+  background-image: linear-gradient(
+    to right,
+    var(--eds-theme-color-background-neutral-medium) 50%,
+    var(--eds-theme-color-background-neutral-subtle) 65%,
+    var(--eds-theme-color-background-neutral-medium) 80%
+  );
+  background-size: 500% 100%;
+}
+
+.skeleton--is-animating {
+  animation: bg-wipe 2s ease-out infinite;
+}
```

Comment:
Will `prefers-reduced-motion` be determined via JS?

Reply:
Easiest solution is probably
```
  @media screen and (prefers-reduced-motion) {
    animation: none;
  }
```

Reply:
Added this, and it can be tested by toggling the setting in the OS

![Screen Shot 2023-01-03 at 09 10 34 ](https://user-images.githubusercontent.com/3056447/210384934-96772ec4-65eb-46ef-bf48-e106a307f2fb.png)